### ISSUE TITLE
[Feature] Add search ability

### ISSUE THREAD

**Issue by accountForIssues** [lolo-io/OneList#8]
Instead of having just an add field, it could be a add/search field so that when a user enters text, if it's found, results are displayed so the user can mark it as complete or incomplete. If it's not found, it gets added to the list.

**Comment by lolo-io** [comment]
Might be a good idea for a future release :)

**Comment by clach04** [comment]
Agreed, right now it's too easy to add duplicate items to long lists. Apps with similar UI/UX that have this feature 
  * https://github.com/dsoulayrol/android-sholi
  * https://github.com/openintents/shoppinglist
